src/Pos/WorkMode.hs

```diff
@@ -149,8 +108,8 @@ type RawRealMode ssc =
     SscHolder ssc (
     NtpSlotting (
     SlottingHolder (
-    ContextHolder ssc (
     DBHolder (
+    ContextHolder ssc (
```

Comment:
`MonadReader` from `ether` has default behavior that it tries to match the tag from `MonadReader` and `ReaderT`, commit to the instance in case of match or proceed to the inner transformer otherwise.

I have overriden this behavior for `ContextHolder` but not `DBHolder`, which is why `ContextHolder` has to be the innermost `ReaderT` of all.